<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>314</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtChat</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPlainTextEdit" name="chatHistoryEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>291</width>
      <height>321</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="chatEdit">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>340</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="sendButton">
    <property name="geometry">
     <rect>
      <x>264</x>
      <y>340</y>
      <width>41</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>发送</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>314</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionusername">
   <property name="text">
    <string>username</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
